Src/MoneyFox.Ui/Common/Groups/DateListGroupCollection.cs

```diff
@@ -31,10 +31,12 @@ public DateListGroupCollection(string key, string subtitle, RelayCommand<T>? ite
     /// </summary>
     /// <param name="key">The key for this group.</param>
     /// <param name="itemClickCommand">The command to execute on click</param>
-    private DateListGroupCollection(string key, RelayCommand<T>? itemClickCommand = null)
+    /// <param name="date">The date for this group.</param>
+    private DateListGroupCollection(string key, RelayCommand<T>? itemClickCommand = null, DateTime? date = null)
```

Comment:
@kg8z  is this still used? seems as if this is a left over of the previous implementation.

Reply:
Reading this again I would like to clarify that I mean the `date` parameter :)

Reply:
@NPadrutt  - The date parameter in the constructor is still needed. Prior to my changes, the DateListGroupCollection class didn't contain a date property. It only had a Key (which is a String representation of the date) and a Subtitle (also a String) property. I could have omitted the additional date parameter in the constructor and instead performed a String to Date conversion on the Key in the Get accessor for the property, but this seemed like an expensive operation compared to simply passing an additional variable in the constructor. And because the class uses Generics, the safest way for me to retrieve the date is via the getSortKey delegate, which gets passed by the calling routine. Let me know if you think there's a more elegant way for me to obtain the date. Thanks.

Reply:
Ah, I see. That makes sense. Thanks :)